.golangci.yml

```
@@ -0,0 +1,284 @@
+  skip-dirs:
+    - bin
+    - vendor
+    - .git
+    - coverage
+    - test_results
+
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+    check-blank: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)